sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/visitors/outer/ToJsonVisitor.java

```diff
@@ -143,6 +144,53 @@ void process(DBSPSimpleOperator operator) {
         this.builder.decrease().append("}");
     }
 
+    void processWithError(DBSPOperatorWithError operator) {
```

Comment:
this is how you emit json? no library for this stuff in java?

Reply:
I tried to use some reflection-based libraries, but the code to control what is serialized, where and how was much worse. So I am essentially serializing everything manually, and this way I control what properties are serialized and how